bundle/edu.gemini.p2checker/src/main/scala/edu/gemini/p2checker/rules/ghost/GhostRule.scala

```diff
@@ -30,7 +32,25 @@ object GhostRule extends IRule {
         t <- env.getAsterism.allTargets
         c <- t.coords(ctx.getSchedulingBlockStart.asScalaOpt.map(Long2long))
         if Coordinates.difference(base, c).distance.toArcsecs > GhostScienceAreaGeometry.radius.toArcsecs
-      } problems.addError(GhostRule.Prefix + "CoordinatesOutOfRange", String.format(GhostRule.CoordinatesOutOfRange, t.name), toc)
+      } problems.addError(GhostRule.Prefix + "CoordinatesOutOfRange", String.format(CoordinatesOutOfRange, t.name), toc)
+
+      problems
+    }
+
+    private val CoordinatesOutOfRange: String = "The coordinates for %s are out of range at the base position."
+  }
+
+  object OffsetsRule extends IRule {
+    override def check(elements: ObservationElements): IP2Problems = {
+      val problems = new P2Problems()
+
+      for {
+        ctx <- elements.getObsContext.asScalaOpt
+        env <- Option(ctx.getTargets)
+        if (env.getAsterism.asterismType == AsterismType.GhostSingleTarget && ctx.getSciencePositions.size > 1) ||
+          (env.getAsterism.asterismType != AsterismType.GhostSingleTarget && !OffsetUtil.allOffsetPosLists(elements.getObservationNode).isEmpty)
```

Comment:
I suppose you could add an offset iterator with a single "offset" at (0, 0) and then you'd trigger this error, but it's probably not worth worrying about.